impeller/entity/contents/content_context.h

```diff
@@ -387,6 +394,16 @@ class ContentContext {
 
   bool IsValid() const;
 
+  /// This setting does two things:
+  /// 1. Enables clipping with the depth buffer, freeing up the stencil buffer.
+  ///    See also: https://github.com/flutter/flutter/issues/138460
+  /// 2. Switches the generic tessellation fallback to use stencil-then-cover.
+  ///    See also: https://github.com/flutter/flutter/issues/123671
+  ///
+  // TODO(bdero): Remove this setting once StC is fully de-risked
+  //              https://github.com/flutter/flutter/issues/123671
+  static constexpr bool kEnableStencilThenCover = false;
```

Comment:
I like the `if constexpr`!

Reply:
Woo! I'll probably use this pattern for other huge feature migrations in the future. ContentContext turned out to be a good place for it because everything in the universe depends on it. ;)